rdr_service/api/nph_participant_biobank_order_api.py

```diff
@@ -104,3 +104,19 @@ def patch(self, nph_participant_id, rdr_order_id):
         except BadRequest as bad_request:
             logging.error(bad_request.description, exc_info=True)
             return construct_response(order), 400
+
+
+class DlwDosageApi(BaseApi):
+
+    def __init__(self):
+        super().__init__(DlwDosageDao())
+
+    @auth_required(RDR_AND_HEALTHPRO)
+    def post(self, nph_participant_id):
+        resource = self.get_request_json()
+        m = self._get_model_to_insert(resource, nph_participant_id)
+        result = self._do_insert(m)
+
+        log_api_request(log=request.log_record, model_obj=result)
+        self._archive_request_log()
+        return self._make_response(result)
```

Comment:
I had `super().post(participant_id=nph_participant_id)` here, but for some reason the 2 lines in base_api's `post` mentioned below 👇  were causing issues.
```
if GAE_PROJECT == 'localhost':
                if ov_id:
                    onsite_id_verification_build(ov_id)
                else:
                    bq_participant_summary_update_task(participant_id)   <---
                    rebuild_participant_summary_resource(participant_id)  <---
```

Reply:
How you have it is fine. Those lines are for PDR and are not necessary for this API.